src/request/src/network.js

```diff
@@ -179,6 +179,32 @@ export class KinveyRequest extends NetworkRequest {
     this.trace = options.trace === true;
   }
 
+  static executeShort(options, client, dataOnly = false) {
```

Comment:
My reasoning was that every time an HTTP request was made, the same 10 lines were written everywhere and then after every `execute()`, there was a `.then(r => r.data)`. I agree two such methods are a bit much. I just wanted to separate the "many assumptions" method from the "execute" method.

I agree the consumer should have the option to use anything and everything, but I think there should be a shorter version. I think in most cases (like 98%) you don't want the statusCode of the successful response, or want to use auth different from "session". Saving one line on a method you call 100 times is a big gain - improves readability, reduces chances of a typo.

Maybe we could merge `execute()` and `executeShort()` with some adjustments on the defaults?